src/tools/auth0/handlers/brandingTemplates.js

```diff
@@ -0,0 +1,82 @@
+import DefaultHandler from './default';
+import constants from '../../constants';
+import log from '../../../logger';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    templates: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          template: { type: 'string' },
+          body: { type: 'string' }
+        }
+      }
+    }
+  }
+};
+
+export default class BrandingTemplatesHandler extends DefaultHandler {
+  constructor(options) {
+    super({
+      ...options,
+      type: 'branding'
+    });
+  }
+
+  async getType() {
+    // in case client version does not support branding
+    if (!this.client.branding || typeof this.client.branding.getUniversalLoginTemplate !== 'function') {
+      return {};
```

Comment:
Is it correct to return an empty object here? I'm mostly asking this out loud, not sure myself off the top of my head.

Reply:
Personally, I think throwing an exception might be a better choice, to make things less mysterious, however, this is the pattern used already in other handlers for missing client functionality, so I went the same route. I think if you want to change it, you might want to consider it a breaking change and do it in a separate PR.

Reply:
There are some hidden levers behind the scenes that control the type of tenant that's provisioned to a customer, so in certain cases, some customers tenants may not have certain features. I don't know if that's applicable for this feature specifically, but it's certainly possible. For that reason, fault-tolerance is preferred in this situation.

I'm mostly trying to understand the contract imposed by these handlers and if an empty object adequately describes a n empty configuration. But that's a discussion for another time. Since it's an existing pattern let's keep what you have here.